addons/website_sale/controllers/main.py

```diff
@@ -89,55 +87,49 @@ def __init__(self, path='', **args):
     def __call__(self, path=None, **kw):
         if not path:
             path = self.path
-        for k,v in self.args.items():
-            kw.setdefault(k,v)
+        for k, v in self.args.items():
+            kw.setdefault(k, v)
         l = []
-        for k,v in kw.items():
+        for k, v in kw.items():
             if v:
                 if isinstance(v, list) or isinstance(v, set):
-                    l.append(werkzeug.url_encode([(k,i) for i in v]))
+                    l.append(werkzeug.url_encode([(k, i) for i in v]))
                 else:
-                    l.append(werkzeug.url_encode([(k,v)]))
+                    l.append(werkzeug.url_encode([(k, v)]))
         if l:
             path += '?' + '&'.join(l)
         return path
 
 
 def get_pricelist():
-    return request.website.get_current_pricelist()
+    return request.website.pricelist_id
```

Comment:
Why do you change this? Don't forget that this should only contain migration, not optimisations or improvements.